src/test/java/guitests/ContextMenuTests.java

```diff
@@ -97,4 +105,23 @@ public void test3() {
         sleep(EVENT_DELAY);
     }
 
+    private void markAllReadUnreadHelper(boolean isTestingRead){
+        click("#dummy/dummy_col0_9");
+        rightClick("#dummy/dummy_col0_9");
+        if (isTestingRead){
+            click("Mark all below as read");
+        } else {
+            click("Mark all below as unread");
+        }
+        PlatformEx.waitOnFxThread();
```

Comment:
Removed this. I just added this to see why the test was failing on the CI but passing every time locally. Still have to debug this further.